drivers/phy/marvell/phy-mvebu-cp110-comphy.c

```diff
@@ -80,6 +80,8 @@
 #define MVEBU_COMPHY_TX_SLEW_RATE(n)		(0x974 + (n) * 0x1000)
 #define     MVEBU_COMPHY_TX_SLEW_RATE_EMPH(n)	((n) << 5)
 #define     MVEBU_COMPHY_TX_SLEW_RATE_SLC(n)	((n) << 10)
+#define MVEBU_COMPHY_TX_AMP_EMP(n)		(0x978 + (n) * 0x1000)
+#define     MVEBU_COMPHY_TX_EMPH_AMP_FORCE		BIT(8)
```

Comment:
why abbreviations are in different sequence? maybe 
```suggestion
#define     MVEBU_COMPHY_TX_AMP_EMP_FORCE		BIT(8)
```

Reply:
this is the name from the datasheet.

Reply:
I will fix one of them according to the order in the datasheet. Thanks.